main.js

```diff
@@ -73,10 +74,20 @@ const askForSecrets = async () => {
 		}
 		localConf.username = mc_username;
 	}
-	if(config.get("discordBot") && !config.has("BotToken")) {
+	if(config.get("discordBot") && !(config.has("BotToken") && config.has("Bot"))) {
 		canSave = true;
-		discordBotToken = await promisedQuestion("BotToken, leave blank if not using discord []: ");
-		localConf.BotToken = discordBotToken;
+		let usebot = (await promisedQuestion("Do you want to use a discord bot? Y or N: [Y]")).toLowerCase() === 'y';
+		if (usebot) {
+			discordBotToken = await promisedQuestion("BotToken: ");
+			localConf.BotToken = discordBotToken;
+			Bot = "2";
+			localConf.Bot = Bot;
+		}	else {
+			Bot = "1";
+			localConf.Bot = Bot;
+			localConf.BotToken = "0";
+		}
+		
```

Comment:
Please change this logic to make better use of the existing config options like I described in a previous comment